### ISSUE TITLE
General Error Auth_OpenID_RAND_SOURCE

### ISSUE THREAD

**Issue by BloodWolf89** [mjimeyg/alternate-login#16]
I am trying to log in to my site with openID using my wargaming.net ID.. They give you an open ID.. Here is the info from them on openID
"Authorized Third-Party Websites

You can use your Wargaming.net OpenID to access fan resources and partner websites. For more on Wargaming.net OpenID and managing third-party websites see this article.
Your Wargaming.net OpenID:
https://na.wargaming.net/id/XXXXXXXX" 

*NOTE: I blocked out my real openID!

I am able to use it on any openID websites but my own. Is there any way to add in an open ID selection just for Wargaming.net?  

Also, I can not see the settings when I go to edit a user. the openID settings are blank.

[phpBB Debug] PHP Warning: in file [ROOT]/includes/openid/Auth/Yadis/XRDS.php on line 439: Parameter 1 to filter_MatchesAnyOpenIDType() expected to be a reference, value given

General Error
Define Auth_OpenID_RAND_SOURCE as null to continue with an insecure random number generator.

**Comment by mjimeyg** [comment]
Are you able to log in to your site with other OpenIDs such as google or yahoo?

Do you get the same error if you try your account on http://consof.com/forums

**Comment by BloodWolf89** [comment]
No, It does not appear to work on that site either. If you want to test it, go to here: https://na.wargaming.net/

and make an account. Its free. Let me know when it works.

**Comment by mjimeyg** [comment]
I made an account and checked the wargaming wiki but it says you have to enter:

http://na.wargaming.net/id/

as the url.

I did that and it took me through to wargaming and asked me to sign in and then returned me to my forum for registration.
